atc/syslog/drainer.go

```diff
@@ -99,10 +94,12 @@ func (d *drainer) drainBuild(logger lager.Logger, build db.Build, syslog *Syslog
 				return err
 			}
 
-			payload := log.Payload
-			tag := build.TeamName() + "/" + build.PipelineName() + "/" + build.JobName() + "/" + build.Name() + "/" + string(log.Origin.ID)
-
-			err = syslog.Write(d.hostname, tag, time.Unix(log.Time, 0), payload)
+			err = syslog.Write(
+				d.hostname,
+				build.SyslogTag(log.Origin.ID),
```

Comment:
Hmm, yeah I think this will probably have to change more in the future. I kept it backwards-compatible for job builds, at least.

Given that we don't document the tag format, and it has only changed for one-off builds (which I don't think people care about syslogs for?), it I think it should be OK to change, but maybe @scottietremendous has more context on how customers use this, since I think it was originally a customer driven request.

FWIW I think this could be argued as a bugfix.